torchvision/prototype/models/googlenet.py

```diff
@@ -14,14 +14,13 @@
 
 
 class GoogLeNetWeights(Weights):
-    ImageNet1K_Community = WeightEntry(
+    ImageNet1K_TFV1 = WeightEntry(
```

Comment:
This was previously incorrectly tagged as community contribution while in reality the weights were ported from TF. Proof: https://github.com/pytorch/vision/pull/678#issuecomment-470318912